fuel-indexer/src/api.rs

```diff
@@ -60,6 +60,10 @@ pub async fn query_graph(
     }
 }
 
+pub async fn health_check() -> StatusCode {
+    StatusCode::OK
```

Comment:
seems fine, although it could be nice to have a json response here like:

```
{
    "uptime": 12345,
    "fuel-core-status": "healthy"
    "database-status": "healthy"
}
```